docs/api/ref/schemas/product_nutrition.yaml

```diff
@@ -11,15 +11,38 @@ properties:
       By doing so, the no_nutrition_data field takes the value "on".
       This case is frequent (thousands of products).
     example: "on"
+  nutrition_data_per:
+    type: string
+    enum:
+    - serving
+    - 100g
+    description: |
+      The nutrition data on the package can be per serving or per 100g.
+
+      This is essential to understand if `<nutrient>_value` and `<nutrient>` 
+      values in `nutriments` applies for a serving or for 100g.
```

Comment:
Can we add something like: "When writing products, this setting applies to all existing nutrients values for the product, not only the nutrient values sent in the write request. So it should not be changed unless all nutrients values are provided with values that match the nutrition_data_per field."